scripts/nxutils.sh

```diff
@@ -1,54 +0,0 @@
-#!/bin/bash
-#
-# Convenient functions for use on Nuxeo projects.
-#
-# (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and contributors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# Contributors:
-#   Kevin Leturc
-#
-
-function _retrieve_modules {
```

Comment:
I did not find any reference to `_retrieve_modules` function, only to `_execute_on_modules` which has not changed.